ge_releaser/changelog.py

```diff
@@ -24,23 +24,20 @@ def __init__(self, pr: PullRequest) -> None:
         with open(GxFile.TEAMS) as f:
             teams = f.read().strip()
 
-        if pr.title[0] == "[":
-            try:
-                type_, self.desc = re.match(r"\[([a-zA-Z]+)\] ?(.*)", pr.title).group(
-                    1, 2
-                )
-                type_ = type_.upper()
-                self.pr_type = (
-                    ChangelogCategory[type_]
-                    if type_ in ChangelogCategory.__members__
-                    else ChangelogCategory.MAINTENANCE
-                )
-            except AttributeError:
-                print(f"Couldn't parse PR title: {pr.title}")
-                return
```

Comment:
Previously was a nullable return - should always return a value (defaulting to `[MAINTENANCE]` if a parsing error occurs)